cvat/apps/dataset_manager/bindings.py

```diff
@@ -579,9 +579,7 @@ def _init_frame_info(self):
                 } for frame in range(task.data.size)})
             else:
                 self._frame_info.update({(task.id, self.rel_frame_id(task.id, db_image.frame)): {
-                    "path": mung_image_name(db_image.path,
-                        filter(lambda frame: frame["subset"] == task.subset ,self._frame_info.values())
-                    ),
+                    "path": mangle_image_name(db_image.path, self._frame_info.values(), task.subset),
```

Comment:
I see this dict is built in O(task_size ^ 2) time. Maybe, we can find a better approach?